tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -0,0 +1,407 @@
+    "azure-schemaregistry",
+    "azure-schemaregistry-avroencoder",
+    "azure-search-documents",
+    "azure-servicebus",
+    "azure-storage-blob",
+    "azure-storage-blob-changefeed",
+    "azure-storage-file-datalake",
+    "azure-storage-file-share",
+    "azure-storage-queue",
+    "azure-synapse-accesscontrol",
+    "azure-synapse-artifacts",
+    "azure-synapse-managedprivateendpoints",
+    "azure-synapse-monitoring",
+    "azure-synapse-spark",
+    "azure-data-tables",
+    "azure-ai-textanalytics",
+    "azure-ai-translation-document",
+    "azure-messaging-webpubsubservice",
+]
+
+# omit package from running verifytypes checks
+VERIFYTYPES_OPT_OUT = [
+    "azure-agrifood-farming",
+    "azure-ai-anomalydetector",
+    "azure-appconfiguration",
+    "azure-appconfiguration-provider",
+    "azure-security-attestation",
+    "azure-batch",
+    "azure-ai-language-conversations",
+    "azure-ai-language-questionanswering",
+    "azure-communication-chat",
+    "azure-communication-email",
+    "azure-communication-identity",
+    "azure-communication-jobrouter",
+    "azure-communication-networktraversal",
+    "azure-communication-phonenumbers",
+    "azure-communication-rooms",
+    "azure-communication-sms",
+    "azure-confidentialledger",
+    "azure-containerregistry",
+    "azure-core",
+    "azure-mgmt-core",
+    "azure-core-experimental",
+    "azure-core-tracing-opencensus",
+    "azure-core-tracing-opentelemetry",
+    "azure-cosmos",
+    "azure-developer-devcenter",
+    "azure-iot-deviceupdate",
+    "azure-digitaltwins-core",
+    "azure-eventgrid",
+    "azure-eventhub",
+    "azure-eventhub-checkpointstoreblob",
+    "azure-eventhub-checkpointstoreblob-aio",
+    "azure-eventhub-checkpointstoretable",
+    "azure-ai-formrecognizer",
+    "azure-identity",
+    "azure-keyvault-administration",
+    "azure-keyvault-certificates",
+    "azure-keyvault-keys",
+    "azure-keyvault-secrets",
+    "azure-developer-loadtesting",
+    "azure-maps-geolocation",
+    "azure-maps-render",
+    "azure-maps-route",
+    "azure-maps-search",
+    "azure-ai-metricsadvisor",
+    "azure-mixedreality-authentication",
+    "azure-ai-ml",
+    "azure-iot-modelsrepository",
+    "azure-monitor-ingestion",
+    "azure-monitor-opentelemetry-exporter",
+    "azure-monitor-query",
+    "azure-ai-personalizer",
+    "azure-purview-administration",
+    "azure-purview-catalog",
+    "azure-purview-scanning",
+    "azure-mixedreality-remoterendering",
+    "azure-schemaregistry",
+    "azure-schemaregistry-avroencoder",
+    "azure-search-documents",
+    "azure-servicebus",
+    "azure-storage-blob",
+    "azure-storage-blob-changefeed",
+    "azure-storage-file-datalake",
+    "azure-storage-file-share",
+    "azure-storage-queue",
+    "azure-synapse-accesscontrol",
+    "azure-synapse-artifacts",
+    "azure-synapse-managedprivateendpoints",
+    "azure-synapse-monitoring",
+    "azure-synapse-spark",
```

Comment:
yesterday I added   "azure-ai-textanalytics" to this list since I found out I wasn't ready for this check. Doesn't look like it got carried over in the move.

Reply:
I must have screwed up the merge. Fixing.